homeassistant/components/lightwave/sensor.py

```diff
@@ -0,0 +1,67 @@
+"""Support for LightwaveRF TRV - Associated Battery."""
+import logging
+
+from homeassistant.const import CONF_NAME, DEVICE_CLASS_BATTERY
+from homeassistant.helpers.entity import Entity
+
+from . import CONF_SERIAL, LIGHTWAVE_LINK
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Find and return battery."""
+    if not discovery_info:
+        return
+
+    batt = []
+
+    lwlink = hass.data[LIGHTWAVE_LINK]
+
+    for dummy, device_config in discovery_info.items():
+        name = device_config[CONF_NAME]
+        serial = device_config[CONF_SERIAL]
+        batt.append(LightwaveBattery(name, lwlink, serial))
+
+    async_add_entities(batt)
+
+
+class LightwaveBattery(Entity):
+    """Lightwave TRV Battery."""
+
+    def __init__(self, name, lwlink, serial):
+        """Initialize the Lightwave Trv battery sensor."""
+        self._name = name
+        self._state = None
+        self._lwlink = lwlink
+        self._serial = serial
+        self._device_class = DEVICE_CLASS_BATTERY
+        self._unit_of_measurement = "%"
```

Comment:
Please import and use the `UNIT_PERCENTAGE` constant from homeassistant/const.py and return it under the property instead of adding another attribute to this class.

Reply:
done